File: golang/2497/2497.go
package main
import "fmt"
func main(){
	aux := 0
	for true{
		var entrada, cont int
		fmt.Scanln(&entrada)
		if entrada == -1{
			break
		}
		aux += 1
		cont = 0
		for true{
			if entrada <= 1{
				break
			}
			cont += 1
			entrada -= 2
		}
		fmt.Printf("Experiment %d: %d full cycle(s)\n", aux, cont)
	}
}

File: Python/2497.py
aux = 0
while True:
    entrada = int(input())
    if entrada == -1:
        break
    aux += 1
    cont = 0
    while entrada > 1:
        cont += 1
        entrada -= 2
    print("Experiment {}: {} full cycle(s)".format(aux, cont))